mantle/kola/tests/podman/podman.go

```diff
@@ -154,7 +154,7 @@ func podmanWorkflow(c cluster.TestCluster) {
 			return nil
 		}
 
-		if err := util.Retry(6, 10*time.Second, podIsRunning); err != nil {
+		if err := util.RetryUntilTimeout(15*time.Minute, 5*time.Minute, podIsRunning); err != nil {
```

Comment:
does this mean it will only try every 5 minutes? I think checking more frequently (like 30s or 1m) would be more ideal.

Previously it was only trying 6 times at an interval of 10s (i.e. 1 minute until timeout/failure). Now if it fails the first time we are waiting 5 minutes for this test to try again.